frontend/src/App.js

```diff
@@ -15,10 +17,19 @@ function App() {
           <Routes>
             <Route path="/" element={<Navigate to="/blog" />} />
             <Route path="/blog" element={<Blog />} />
-            <Route path="/blog/createArticle" element={<CreateArticle />} />
+            <Route
+              path="/blog/createArticle"
+              element={user ? <CreateArticle /> : <Navigate to="/login" />}
+            />
             <Route path="/blog/:id" element={<Article />} />
-            <Route path="login" element={<Login />} />
-            <Route path="signup" element={<Signup />} />
+            <Route
+              path="login"
+              element={!user ? <Login /> : <Navigate to="/" />}
+            />
+            <Route
+              path="signup"
+              element={!user ? <Signup /> : <Navigate to="/" />}
```

Comment:
There are two buttons to go to home so?

Reply:
No, actually this line of code tells that if the user is logged in, the route `iscsc.fr/blog/signup` will redirect to `iscsc.fr/` (which at the moment redirects to `iscsc.fr/blog`)

Reply:
Ok tks!